src/main/java/edu/cornell/kfs/module/purap/document/CuPurchaseOrderAmendmentDocument.java

```diff
@@ -121,7 +121,7 @@ public Set<Person> getAllPriorApprovers() throws WorkflowException {
     // KFSUPGRADE-339
     protected boolean isContractManagementReviewRequired() {
         final ParameterService parameterService = SpringContext.getBean(ParameterService.class);
-        final KualiDecimal automaticPurchaseOrderDefaultLimit = new KualiDecimal(parameterService.getParameterValueAsString(RequisitionDocument.class, PurapParameterConstants.AUTOMATIC_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT));
+        final KualiDecimal automaticPurchaseOrderDefaultLimit = new KualiDecimal(parameterService.getParameterValueAsString(RequisitionDocument.class, PurapParameterConstants.APO_LIMIT));
```

Comment:
Should this be modified to use the namespace-component-parameter variant instead of the classname-parameter variant? I thought this parameter was updated to use the "PurchaseOrder" component.